cognite/src/dto/data_modeling/common.rs

```diff
@@ -79,6 +79,16 @@ pub enum TaggedContainerReference {
     Container(ItemId),
 }
 
+impl TaggedContainerReference {
+    /// Create a new tagged container reference.
+    pub fn new(space: impl Into<String>, external_id: impl Into<String>) -> Self {
+        Self::Container(ItemId {
+            space: space.into(),
+            external_id: external_id.into(),
+        })
+    }
+}
```

Comment:
Nitpick: I am wondering if `&str` is not more flexible than `Into<String>` here. 

```suggestion
    pub fn new(space: &str, external_id: &str) -> Self {
        Self::Container(ItemId {
            space: space.to_string(),
            external_id: external_id.to_string(),
        })
    }
}
```

Reply:
`&str` implements `Into<String>` so I do not see how that could be more flexible?